app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <application
         android:name=".MifosSelfServiceApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_icon"
```

Comment:
@Pritom14 We have decided to use Mifos globe icon for now. check the mifos_logo.png in drawable. You need to separate the globe icon and make them hdpi, xhdpi, etc sizes and put them in mipmap folder.